examples/cra-kitchen-sink/src/image-storyshots.test.js

```diff
@@ -1,10 +1,31 @@
 import initStoryshots, { imageSnapshot } from '@storybook/addon-storyshots';
 import path from 'path';
+import fs from 'fs';
 
-initStoryshots({
-  suite: 'Image snapshots',
-  framework: 'react',
-  storyKindRegex: /ImageSnapshot$/,
-  configPath: path.join(__dirname, '..', '.storybook'),
-  test: imageSnapshot({ port: 9010 }),
-});
+// We do screenshots against the static build of the storybook.
+// For this test to be meaningful, you must build the static version of the storybook *before* running this test suite.
+const pathToStorybookStatic = path.join(__dirname, '..', 'storybook-static');
+
+const skip = process.env.SKIP_IMAGE_SNAPSHOTS;
+
+if (skip) {
+  // Do nothing.
+} else if (!fs.existsSync(pathToStorybookStatic)) {
+  console.error(
+    'You are running image snapshots without having the static build of storybook. Please run "yarn run build-storybook" before running tests.'
+  );
+} else {
+  initStoryshots({
+    suite: 'Image snapshots',
+    framework: 'react',
+    storyKindRegex: /ImageSnapshot$/,
+    configPath: path.join(__dirname, '..', '.storybook'),
+    test: imageSnapshot({
+      storybookUrl: `file://${pathToStorybookStatic}`,
```

Comment:
Will it work on windows? @igor-dv can you test please?

Reply:
Good point, let's see what he says

Reply:
I'll check it today 👌

Reply:
Checked on windows. Works 👍